zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/handlers/PostImporterQueueFromIncidentHandlerTest.java

```diff
@@ -40,11 +42,14 @@ void testGetEntityType() {
     assertThat(underTest.getEntityType()).isEqualTo(PostImporterQueueEntity.class);
   }
 
-  @Test
-  void shouldHandleRecord() {
+  @ParameterizedTest
+  @EnumSource(
+      value = IncidentIntent.class,
+      names = {"CREATED", "MIGRATED", "RESOLVED"})
+  void shouldHandleRecordWithSupportedIntent(final IncidentIntent intent) {
```

Comment:
🔧 can we also have a test for the `shouldNotHandle...` in this class? i.e. for `RESOLVE`